Uppsala/UppEt7.xml

```diff
@@ -0,0 +1,336 @@
+                                ክፍል፡ ቀዳሚ፡ ዘእምተስእሎታት፡ ዜናሆሙ፡ ለአበወ፡ መነኮሳት፡ ግብጻውያን፨
+                                ወጸሐፎ፡ ቅዱስ፡ ፊልክስዩስ፡ ሶርያዊ፡ ኤጲስ፡ ቆጶስ፡ ዘመንበግ፡ 
+                                ጸሎቱ፡ ወበረከቱ፡ የሀሉ፡ ምስለ፡ ገብሩ፡ 
+                                <hi rend="rubric"><del rend="erasure" unit="chars" quantity="6"/>ስ፡</hi> 
+                                ለዓለመ፡ ዓለም፡ አሜን<hi rend="partialRubric">፨</hi>
+                            </incipit>
+                            
+                            <msItem xml:id="ms_i1.1">
+                                <locus from="1ra" to="13ra"/>
+                                <title type="complete">Part 1: Questions 1–16 with answers</title>
+                                <note><bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">105</citedRange></bibl>
+                                    cites the incipit. He
+                                    notes that the colophon of this part (on <locus target="#13ra"/>) mentions that forty questions are missing from
+                                    the beginning (<foreign xml:lang="gez">እምቀዳሚሁ፡</foreign>), concluding that the text has been
+                                    abbreviated.
+                                </note>
+                                <incipit xml:lang="gez">
+                                    በእንተ፡ ዘከመ፡ ይኄይስ፡ ሥርዓተ፡ ብሕትውና፡ እምሥርዓተ፡ ማኅበር<hi rend="partialRubric">፨</hi>
+                                    <hi rend="rubric">ክፍል፡</hi> <hi rend="partialRubric">፩</hi>ይቤሉ፡ 
+                                    አኃው፡ ንወጥን፡ እምጥንቱ፡ ለግብር፡ ሀለዉ፡ ክልኤቱ፡ አኃው፡ 
+                                    ወልደ፡ ብእሲ፡ ነጋዲ፡ ወሶበ፡ አዕረፈ፡ አባሆሙ፡ ወተካፈሉ፡ ርስቶ፡
+                                </incipit>
+                                <explicit xml:lang="gez">
+                                    ወይደሉ፡ እንከ፡ ለነ፡ ከመ፡ ንእመን፡ ዘይትበሀል፡ በእንተ፡ ቅዱሳን፡ እምዜናውያን፡ 
+                                    ጽዱቃውያን<hi rend="partialRubric">፨</hi> ወለእመኒ፡ ኢሀለዉ፡ በአዝማኒነ፡ 
+                                </explicit>
+                                <explicit type="colophon" xml:lang="gez">
+                                    <locus target="#13ra"/>
+                                    ተፈጸመ፡ ክፍል፡ ቀዳሚ፡ በረድኤተ፡ እግዚአብሔር፡ ወነገረ፡ ከመ፡ ተኃጥአ፡ እምቀዳሚሁ፡ 
+                                    <hi rend="partialRubric">፵</hi>ተስእሎታት<hi rend="partialRubric">፨</hi>
+                                </explicit>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.2">
+                                <locus from="13ra" to="18va"/>
+                                <title type="complete">Part 2: Questions 17–34 with answers</title>
+                                <note><bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">105</citedRange></bibl>
+                                    quotes the colophon (on <locus target="#18va"/>) and translates it as follows: "End of the abbreviation of Part 2"
```

Comment:
If it is a colophon, it should be encoded as a colophon (here and in other cases).

Reply:
I checked another manuscript with the same text (DD-048), and it also has the same colophons, so I will encode them as embedded colophons

Reply:
After rereading the guidelines, I think it's possible to argue that this is not a colophon. I will leave them as explicits